specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/keysManagedHsm.json

```diff
@@ -0,0 +1,630 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-11-01",
+    "title": "KeyVaultManagementClient",
+    "description": "The Azure management API provides a RESTful set of web services that interact with Azure Key Vault."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.KeyVault/managedHSMs/{name}/keys/{keyName}": {
+      "put": {
```

Comment:
>put [](http://example.com/codeflow?start=7&length=3)

Why no delete or update operation?

Reply:
This is proxy resource and due to strict permissions requirement delete and patch can only be performed through data plane by data plane admins and not through ARM.